ui/components/component-library/text/text.scss

```
@@ -7,6 +7,13 @@ $text-variants: (
   body: ("xs", "xs-medium", "sm", "sm-medium", "sm-bold", "md", "md-medium", "md-bold", "lg-medium"),
 );
 
+// Phase 1: Brand evolution font families
```

nit: I caused some confusion by using the "Phase" language. It seems DS and Wallet UX aren't aligned on what's included in each phase, so it might be safer to remove it.
```suggestion
// Brand evolution font families
```